examples/webcam_face_pose_ex.cpp

```diff
@@ -27,61 +27,85 @@
     2011.  SSE4 is the next fastest and is supported by most current machines.  
 */
 
-#include <dlib/opencv.h>
```

Comment:
This is a great example.  Put it in a separate example though.  Call it `webcam_ffmpeg_fae_pose_ex.cpp` or something like that and keep the opencv one.  That way people who want to use opencv can still have a little example showing them how to use dlib and opencv together.

Reply:
done